README.rst

```diff
@@ -229,15 +229,15 @@ VERIFYING_KEY
   ``VERIFYING_KEY`` setting will be ignored and the value of the
   ``SIGNING_KEY`` setting will be used.  If an RSA algorithm has been specified
   by the ``ALGORITHM`` setting, the ``VERIFYING_KEY`` setting must be set to a
-  string which contains an RSA public key.
+  string that contains an RSA public key.
 
 AUDIENCE
-  The audience claim to be included in generated tokens and/or validated in
```

Comment:
Don't think we should change this.  This setting becomes the value of a single "claim" in the JWT payload (the "aud" claim).  Therefore, I don't think it's correct to say that there's more than one claim here.